games-fps/yamagi-quake2/yamagi-quake2-7.42.ebuild

```diff
@@ -0,0 +1,154 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop eutils
+
+CTF_V=1.06
+ROGUE_V=2.06
+XATRIX_V=2.07
+
+DESCRIPTION="Quake 2 engine focused on single player"
+HOMEPAGE="https://www.yamagi.org/quake2/"
+SRC_URI="https://deponie.yamagi.org/quake2/quake2-${PV}.tar.xz
+	ctf? ( https://deponie.yamagi.org/quake2/quake2-ctf-${CTF_V}.tar.xz )
+	rogue? ( https://deponie.yamagi.org/quake2/quake2-rogue-${ROGUE_V}.tar.xz )
+	xatrix? ( https://deponie.yamagi.org/quake2/quake2-xatrix-${XATRIX_V}.tar.xz )"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+client ctf dedicated openal +opengl rogue softrender xatrix"
+REQUIRED_USE="|| ( client dedicated ) client? ( || ( opengl softrender ) )"
+
+COMMON_DEPEND="
+	client? (
+		media-libs/libsdl2[video]
+		openal? ( media-libs/openal )
+		!openal? ( media-libs/libsdl2[sound] )
+		opengl? (
+			media-libs/libsdl2[opengl]
+			virtual/opengl
+		)
+	)
+"
+RDEPEND="${COMMON_DEPEND}
+	client? ( openal? ( media-libs/openal ) )
```

Comment:
Doesn't this already come from inheriting `COMMON_DEPEND`?

Reply:
It's runtime only dependency, so it shouldn't be in `COMMON_DEPEND`